.gitignore

```diff
@@ -15,3 +15,4 @@ app/cache
 composer.phar
 docs/src
 build.properties
+var
```

Comment:
Can you also add an empty gitkeep file in the var/log and var/cache folder, and add that as an exception to the gitignore? This is a workaround for git to create folders, otherwise they may not be created :)